### PR TITLE
[JBTM-3336] CORBA classes are missing in the ArjunaJTS/standalone qui…

### PR DESCRIPTION
**Pull request by zezulka**
https://issues.redhat.com/browse/JBTM-3336

### CODE REVIEW

**Comment by ochaloup** [comment]
TESTIT

File: ArjunaJTS/standalone/pom.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <!--
     JBoss, Home of Professional Open Source
-    Copyright 2014, Red Hat, Inc. and/or its affiliates, and individual
```
Comment (ochaloup) [inline ArjunaJTS/standalone/pom.xml:4]: I think this line should be left with the year the file was created and not changed to `2020`. At least it usual way how we do this.

File: ArjunaJTS/standalone/start.sh [hunk 2]
```diff
@@ -1,6 +1,6 @@
 
 # JBoss, Home of Professional Open Source
-# Copyright 2016, Red Hat, Inc., and others contributors as indicated
+# Copyright 2020, Red Hat, Inc., and others contributors as indicated
```
Comment (ochaloup) [inline ArjunaJTS/standalone/start.sh:3]:
```suggestion
# Copyright 2016, Red Hat, Inc., and others contributors as indicated
```

File: ArjunaJTS/standalone/pom.xml (hunk 1 above)
Comment (mmusgrov) [reply ArjunaJTS/standalone/pom.xml:4]: +1

File: ArjunaJTS/standalone/start.sh (hunk 2 above)
Comment (zezulka) [reply ArjunaJTS/standalone/start.sh]: Thanks for your feedback, I've just removed all date changes from the commit.

**Pull request merged**